nurseconnect_resubscribe.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+
+import argparse
+import requests
+import sys
+
+session = requests.Session()
+http = requests.adapters.HTTPAdapter(max_retries=5)
+https = requests.adapters.HTTPAdapter(max_retries=5)
+session.mount('http://', http)
+session.mount('https://', https)
+
+
+def create_sub(url, token, data):
+    headers = {
+        'Authorization': "Token " + token,
+        'Content-Type': "application/json"
+    }
+    resp = session.post('%ssubscriptions/' % url, headers=headers, json=data)
+    resp.raise_for_status()
+
+
+def sub_exists(url, token, params):
+    headers = {
+        'Authorization': "Token " + token,
+        'Content-Type': "application/json"
+    }
+    resp = session.get('%ssubscriptions/' % url, headers=headers,
+                       params=params)
+    resp.raise_for_status()
+    if resp.json().get('count') > 0:
+        return True
+    return False
+
+
+def get_subs(url, token, params):
+    results = []
+    next_url = '%ssubscriptions/' % url
+    while next_url != "":
+        headers = {
+            'Authorization': "Token " + token,
+            'Content-Type': "application/json"
+        }
+        resp = session.get(next_url, headers=headers, params=params)
+        resp.raise_for_status()
+        results.extend(resp.json().get('results'))
+        next_url = resp.json().get('next')
+    return results
```

Comment:
Do we maybe want this function to be a generator instead? That way we don't have to wait for all the subscriptions before processing them, and we don't have to keep all the subscriptions in memory. eg:

```
while next_url != "":
    ... get page of results ..
    for result in resp.json().get('results'):
        yield result
   ...
```